acousticbrainz/data/__init__.py

```diff
@@ -1,192 +1,14 @@
 import psycopg2
-from acousticbrainz.utils import sanity_check_data, clean_metadata, interpret_high_level
-from acousticbrainz.data.exceptions import NoDataFoundException
-from werkzeug.exceptions import BadRequest, ServiceUnavailable, InternalServerError, NotFound
-from flask import current_app
-from hashlib import sha256
-import json
-import time
 
+# Be careful when importing this before init_connection function is called!
+connection = None
```

Comment:
@alastair, any idea if there's a way to avoid importing `connection` in every function it is used (like I've done it now)?  There must be a better way. Doesn't seem right.